app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1451,6 +1467,10 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
                         createTrackersAnimation()
                     }
                 }
+
+                if (!viewState.isLoading) {
+                    swipeRefreshContainer.isRefreshing = false
+                }
```

Comment:
```suggestion
  swipeRefreshContainer.isRefreshing = viewState.isLoading
```

Reply:
Since this callback is used for other loading operations (e.g. loading a link), we would show the refresh loading indicator in these cases too with the suggested change.